<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>game_rank</class>
 <widget class="QMainWindow" name="game_rank">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>排行榜</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="rank_view">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>411</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>诗词问答排名前100</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>成语消消看排名前100</string>
     </property>
    </item>
   </widget>
   <widget class="QCommandLinkButton" name="exit_bt">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>0</y>
      <width>131</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string>返回主页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="create_shan">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>80</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>男女比例扇形图</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
